2-mandatory/3-bush-berries.js

```diff
@@ -23,6 +23,14 @@
 
 function isBushSafe(berryArray) {
   //Write your code here
+  for (let berry of berryArray){
+    if (berry ==="pink"){
+     return "Bush is safe to eat from";
+    }else{
+     return "Toxic! Leave bush alone!";
+    }
+}
+
```

Comment:
Well Done !! I would say consider using every( ) or some ( ) as you'll need to make sure that the passed array contains only "pink". If it contains  other than "pink" then "Toxic..." should be returned.  Hope that helps.